app/models/ticket.rb

```diff
@@ -18,7 +18,9 @@ def bought?(user)
   end
 
   def paid?(user)
```

Comment:
What about something like this?
`ticket_purchases.find_by(user: user, paid: true).present?`